charts/redpanda/templates/services.nodeport.yaml

```diff
@@ -15,7 +15,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */}}
 {{- $values := .Values }}
-{{- if (include "external-nodeport-enabled" . | fromJson).bool }}
+{{- if (and .Values.external.enabled (eq .Values.external.type "NodePort")) }}
```

Comment:
Why are we bringing this back in? I prefer to have simple booleans inside the templates when possible, leaving the complexity in the named templates.

Reply:
I needed to make a change to the logic on this, but then realized the only place this named template is ever used is in the nodeport service file. So I just included the logic here. I guess it's possible that someday this named template would need to be used in some other file though, and if that's the case it would make sense to keep it around.

But the named template could be brought back in regardless, let me know.

Reply:
Ok, then I'd prefer to leave this change out of this PR. We can discuss it in a separate PR if you feel strongly.